src/inspector_agent.cc

```diff
@@ -448,8 +506,12 @@ bool IsFilePath(const std::string& path) {
 
 class NodeInspectorClient : public V8InspectorClient {
  public:
-  explicit NodeInspectorClient(node::Environment* env, bool is_main)
-      : env_(env), is_main_(is_main) {
+  explicit NodeInspectorClient(
+      node::Environment* env,
+      bool is_main,
+      const std::vector<std::string>& domains_whitelist)
+      : env_(env), is_main_(is_main),
+        domains_whitelist_(domains_whitelist.begin(), domains_whitelist.end()) {
```

Comment:
Ditto about indentation… [here](https://github.com/nodejs/node/blob/f2061930c83cb579410a7f26bed726d6568575e3/CPP_STYLE_GUIDE.md#initialization-lists) is an example from the style guide